src/app/shared/layout/header.component.html

```diff
@@ -12,10 +12,10 @@
     >WATcher v{{ this.getVersion() }}</a
   >
   <span id="view-descriptor" *ngIf="auth.isAuthenticated()" style="margin-left: 10px">
-    ({{ this.getViewDescription(viewService.currentView) }})
+    ({{ this.presetViews[this.filtersService.presetView$.value] }})
   </span>
 
-  <div *ngIf="auth.isAuthenticated() && this.viewService.sessionData.sessionRepo.length > 1">
+  <!--div *ngIf="auth.isAuthenticated() && this.viewService.sessionData.sessionRepo.length > 1">
```

Comment:
Should this redundant code be left in the code base?

Reply:
We are still unsure whether to keep the activity dashboard. Currently, this is the only gateway to the activity dashboard, so I'm afraid it will be more difficult for the next batch of ppl to unhide activity dashboard, if we ever want to develop it.

Reply:
I see. In that case could you add a comment highlighting its significance for other developers who might remove it on instinct?